File: model/src/main/java/com/sreMake/model/system/CasbinRule.java
package com.sreMake.model.system;

import org.babyfish.jimmer.sql.*;
import org.jetbrains.annotations.Nullable;

@Entity
@Table(name = "casbin_rule")
public interface CasbinRule {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    int id();

    String ptype();

    @Nullable
    String v0();

    @Nullable
    String v1();

    @Nullable
    String v2();

    @Nullable
    String v3();

    @Nullable
    String v4();

    @Nullable
    String v5();

}

File: model/src/main/kotlin/com/SReMake/model/system/CasbinRule.kt
package com.SReMake.model.system

import org.babyfish.jimmer.sql.*

@Entity
@Table(name = "casbin_rule")
interface CasbinRule {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Int
    val ptype: String
    val v0: String?
    val v1: String?
    val v2: String?
    val v3: String?
    val v4: String?
    val v5: String?
}